actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -9,14 +9,32 @@ module SubscriptionAdapter
     class PostgreSQL < Base # :nodoc:
       prepend ChannelPrefix
 
+      PayloadTooLargeError = Class.new(StandardError)
+
+      MAX_NOTIFY_SIZE = 7997 # documented as 8000 bytes, but there appears to be some overhead in transit
+      DEFAULT_LARGE_PAYLOADS_TABLE = "action_cable_large_payloads"
+      LARGE_PAYLOAD_PREFIX = "__large_payload:"
+
+      attr_reader :large_payloads_table
+
       def initialize(*)
         super
         @listener = nil
+        @large_payloads_table = @server.config.cable[:large_payloads_table] || DEFAULT_LARGE_PAYLOADS_TABLE
```

Comment:
NOTE: The table name is configurable, but I have not mentioned this in the documentation. Doesn't seem all that relevant. Any thoughts about configurability or the table name in general?

Reply:
I think you should create a model for this, and use it to interact with the underlying table. Compare to things like `ActionText::RichText`. (Then users can implement `table_name` on the model if they need to change the name.)

Reply:
Could do. But that would imply giving up on https://github.com/rails/rails/issues/27214 as we would couple the PostgreSQL adapter to ActiveRecord in a fairly definite way. Note that this issue is currently also linked from the Action Cable guide.

I don't have a strong opinion personally, so feel free to confirm which approach you think we should take, all things considered.

Reply:
Ahh thanks. I hadn’t seen that issue before. It is probably better to wait for input from core team on this.